tools/compile-scenario-glue.cjs

```diff
@@ -0,0 +1,23 @@
+const { Elm } = require("./ScenarioInOriginalGame/ScenarioAPI.js");
+
+const baseAddress = process.argv[2];
+
+if (baseAddress === undefined) {
+    console.error("Must specify base address.");
+    process.exit(1);
+}
+
+try {
+    const app = Elm.ScenarioAPI.init({
+        flags: { elmFlag_baseAddress: baseAddress },
+    });
```

Comment:
I don’t know if it makes sense, but I just realized that it would be possible to pass  `process.argv.slice(2)` to Elm as a flag and have Elm do the argv parsing. If we’d rather have more Elm code than JavaScript code.

Reply:
Fixed in 6b8bce2. See also a44fb89, 58c5fe7, and 9ae7f55.